docs/glossary.md

```diff
@@ -0,0 +1,265 @@
+
+
+#### Account
+
+Accounts allow you to keep separate records of your DCR funds. Transferring DCR across accounts will create a transaction on the blockchain.
+
+#### Agenda voting
+
+See [consensus rules voting](#consensus-rules-voting).
+
+#### Address
+
+The cryptographic hash of a given public key. Addresses are publicly viewable on the blockchain and can be used to send and receive transactions, similar to an email address. 
+
+#### Atom
+
+The smallest unit of Decred currency. One atom is one hundred millionth of a single DCR (0.00000001 DCR).
+
+#### Block
+
+The data structure transactions are bundled into before being written into the distributed ledger. Decred's difficulty adjusts such that new blocks are found every five minutes on average.
+
+#### Block explorer
+
+A tool for inspecting the contents of blocks in a more user-friendly format. 
+
+#### Block header
+
+Metadata at the beginning of a block that defines information about the block. The hash of a block header is the main way of identifying a block. 
+
+#### Block reward
+
+A reward (in DCR) split between PoW miners, ticket holders, and the Treasury for successfully approving a block of transactions. As in Bitcoin, the block reward [decays exponentially](https://docs.decred.org/advanced/inflation/) with block height (length of blockchain). However, Decred’s algorithm interpolates this decay over time so as not to produce market shocks, as typically seen when Bitcoin's difficulty halves. The block reward started at 31.19582664 and it adjusts every 6,144 blocks (approximately 21.33 days) by reducing by a factor of 100/1011.
+
+#### Block voting
+
+In each block, five tickets are called to vote. In addition to votes on any open consensus rule change proposals, each ticket votes to approve or reject the regular transaction tree of the previous block. If a majority of voting tickets vote No, the regular transaction tree of the previous block is rejected and those transactions are returned to the mempool.
+
+#### Command Line Interface (CLI)
+
+An interface a user interacts with via the command line - often referring to a CLI wallet, which is operated using various tools and commands.
+
+#### Confirmations
+
+The number of blocks that have been mined on top of the block when a transaction was made; more confirmations gives greater finality or confidence that the transaction will remain in the Decred blockchain.
```

Comment:
Not quite accurate.  The most recent block is said to have 1 confirmation even though there are no blocks following it (the transactions within have one block confirmation).

Reply:
I googled around for how other glossaries defined this. All define 'Confirmation' (singular). I think this is better. Have taken a stab at a new definition that also includes the meaning of confirmations (plural). 

#### Confirmation

A transaction is confirmed when it has been included in a block that has been verified by the network (added to the blockchain). Once a transaction has been confirmed, it cannot be reversed or double spent. Each additional block added to the blockchain reconfirms all transactions in all previous blocks. The number of times a transaction has been confirmed (i.e. the number of blocks that include the transaction) is used as a measure of confidence that the transaction will remain in the blockchain. For example. a wallet or other service may requre a certain number of confirmations before it considers a transaction to be valid.

Reply:
Removed line about reversed and double spent.

#### Confirmation

A transaction is confirmed when it has been included in a block that has been verified by the network (added to the blockchain). Each additional block added to the blockchain reconfirms all transactions in all previous blocks. The number of times a transaction has been confirmed (i.e. the number of blocks that include the transaction) is used as a measure of confidence that the transaction will remain in the blockchain. For example. a wallet or other service may requre a certain number of confirmations before it considers a transaction to be valid.

Reply:
"(i.e. the number of blocks that include the transaction)" doesn't read correctly to me, as I read that as claiming the transaction is contained in multiple blocks.  Perhaps "the number of blocks added since the block that includes the transaction".

Reply:
Had thought of that language, but as jrick pointed out above, the most recent block is said to have 1 confirmation even though there are no blocks following it, thus introducing quite the word puzzle. I gave up trying to solve this puzzle and just tried to express this as an example. 

#### Confirmation

A transaction is confirmed when it has been included in a block that has been verified by the network (added to the blockchain). Each additional block added to the blockchain reconfirms all transactions in all previous blocks. The number of times a transaction has been confirmed is used as a measure of confidence that the transaction will remain in the blockchain. A wallet or other service may require a certain number of confirmations before it considers a transaction to be valid. For example, if a wallet requires three confirmations to consider a transaction valid, it would require the next two blocks after the block including the transaction to confirm the transaction.